documentation/modules/creating-vddk-image.adoc

```diff
@@ -40,12 +41,11 @@ $ mkdir /tmp/<dir_name> && cd /tmp/<dir_name>
 
 . In a browser, navigate to the link:https://developer.vmware.com/web/sdk/8.0/vddk[VMware VDDK version 8 download page].
 . Select version 8.0.1 and click *Download*.
-
++
 [NOTE]
 ====
 In order to migrate to {virt} 4.12, download VDDK version 7.0.3.2 from the link:https://developer.vmware.com/web/sdk/7.0/vddk[VMware VDDK version 7 download page].
```

Comment:
I'd remove the virt version numbre (4.12).

Reply:
@fabiand 
Previously i was told that for CNV 4.13 and later, we want them to to use VDDK 8.0.1 [line 43], but  anyone still running CNV 4.12 should use VDDK 7.0.3.2 [the note]. Has that advice changed?

Reply:
@ahadas Do we recall why we added the note about migrating to CNV 4.12?  If the note is correct, please let me know. Thanks.

Reply:
I can't recall if this note was added because CDI on OpenShift 4.12 used an old version of nbdkit that didn't support VDDK 8, or we didn't have the capacity to test this so we limited the use of VDDK 8 from the lowest version we tested it on

Reply:
Thanks, @ahadas . @mnecas @fabiand Let me know what you think is best.

Reply:
@fabiand Removed note completely, as per MTV sync 15 January.